services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminConsumptionTask.java

```diff
@@ -492,7 +497,7 @@ private void unSubscribe() {
   private void executeMessagesAndCollectResults() throws InterruptedException {
     lastSucceededExecutionIdMap =
         new ConcurrentHashMap<>(executionIdAccessor.getLastSucceededExecutionIdMap(clusterName));
-    List<Callable<Void>> tasks = new ArrayList<>();
+    this.tasks = new ArrayList<>();
```

Comment:
Good catch! Let me revert this change. It was intend to help with heapdump analysis, I want to check what's in the task list. Since we found the root cause now, I don't really need this change anymore.